gnucash/gtkbuilder/assistant-stock-transaction.glade

```diff
@@ -0,0 +1,941 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.38.2 -->
+<interface>
+  <requires lib="gtk+" version="3.22"/>
+  <object class="GtkAssistant" id="stock_transaction_assistant">
+    <property name="can-focus">False</property>
+    <property name="halign">start</property>
+    <property name="border-width">12</property>
+    <property name="title" translatable="yes">Stock Split Assistant</property>
+    <property name="window-position">center</property>
+    <property name="default-width">500</property>
+    <signal name="cancel" handler="stock_assistant_cancel" swapped="no"/>
+    <signal name="close" handler="stock_assistant_finish" swapped="no"/>
+    <signal name="prepare" handler="stock_assistant_prepare" swapped="no"/>
+    <child>
+      <object class="GtkLabel" id="intro_page_label">
+        <property name="visible">True</property>
+        <property name="can-focus">False</property>
+        <property name="valign">start</property>
+        <property name="margin-start">12</property>
+        <property name="margin-end">12</property>
+        <property name="label" translatable="yes">This assistant will help you record a stock transaction. The transaction type (purchase, sale, dividend, distribution, return of capital, stock split) will determine the transaction splits involved in the transaction.</property>
+        <property name="wrap">True</property>
+      </object>
+      <packing>
+        <property name="page-type">intro</property>
+        <property name="title" translatable="yes">Stock Transaction Assistant</property>
+        <property name="complete">True</property>
+      </packing>
+    </child>
+    <child>
+      <object class="GtkBox" id="transaction_type_page">
+        <property name="visible">True</property>
+        <property name="can-focus">False</property>
+        <property name="margin-start">12</property>
+        <property name="border-width">6</property>
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="GtkLabel" id="transaction_type_page_label">
+            <property name="visible">True</property>
+            <property name="can-focus">False</property>
+            <property name="margin-bottom">6</property>
+            <property name="label" translatable="yes">Select the type of stock activity that you wish to record. The available types depend on whether the current stock balance is positive, nil, or negative (i.e. when shorting stock). The type will determine the component splits.</property>
```

Comment:
`(i.e. when shorting stock)`: while I know `short selling`  neither google translate nor me knows this term.

Reply:
"Shorting a stock" is a slang-ish way of saying "Selling short a stock", that is borrowing the stock from someone and selling it with the intent to buy it back at a lower price and to redeem the loan.